tests/static/js/jqunit/jqunit-binder-checkbox.js

```diff
@@ -0,0 +1,184 @@
+/* globals fluid */
```

Comment:
These files should have "test" somewhere in their filename. Our convention, for what its worth, is for a JS file named XxxxTests.js (camel case) to be associated with an html file named Xxxxx-test.html (Infusion style) or XxxxxTest.html (GPII style)